### PR TITLE
LPS-123544 obtain the document via title or via filename if exist

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java [hunk 1]
```diff
@@ -445,7 +445,19 @@ else if (pathArray.length == 3) {
 					0, fileName.indexOf(StringPool.QUESTION));
 			}
 
-			return DLAppServiceUtil.getFileEntry(groupId, folderId, fileName);
+			try {
+				return DLAppServiceUtil.getFileEntryByFileName(
```
Comment (4lejandrito) [inline portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java:449]: Should we invert the order of these operations?

I understand this new call is a special case when the fileName and fileEntry are different.

Comment (AliciaGarciaGarcia) [reply portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java:449]: With the new changes, the filename will be the prefered "name" to use to search the file.
And in some of the old cases, the title is the same that the fileName so with the new method we will get the FileEntry, so  less calls wil be, that is why I changed the order.

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**